packages/story-editor/src/app/media/utils/test/useProcessMedia.js

```diff
@@ -110,12 +111,21 @@ function setup() {
   const storyContextValue = {
     actions: { updateElementsByResourceId },
   };
+
+  const configState = {
+    capabilities: {
+      hasUploadMediaAction: true,
+    },
+  };
```

Comment:
Why is this change to the test now needed? I don't see any use of the config provider in the `useProcessMedia` hook.

Reply:
I believe it's because I imported and am using `useFFMEG` inside of useProcessMedia.

Reply:
Oh, I see! 💡 

That's a great point actually. Currently all our FFmpeg usage is centralized in `useMediaUploadQueue` so we can prevent simultaneous transcoding processes. (Otherwise FFmpeg processes could exhaust your CPU easily)

See how `useMediaUploadQueue` handles `muteVideo` and `trimData` today. We should do the same for cropping.